go.mod

```diff
@@ -1,20 +1,17 @@
-module github.com/sourcegraph/deploy-sourcegraph
+module sourcegraph/deploy-sourcegraph
 
-go 1.16
+go 1.21.5
```

Comment:
Does this need bumping too? https://github.com/sourcegraph/deploy-sourcegraph/blob/master/.github/workflows/update-tags.yml#L17

Reply:
Ah good catch. Might as well update it!